www/index.html

```diff
@@ -78,7 +78,6 @@
 </script>
 
 <script src="boot.js"></script>
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Promise,Array.from,Array.prototype.find,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Math.sign,Intl.~locale.en,Intl.~locale.id,Intl.~locale.ru,Intl.~locale.fr&unknown=polyfill"></script>
```

Comment:
You need to include also all the locales. Which are not that small.
Also need to include all polyfills, fetch, Promise, Array.from, Array.find etc. There are a dozen of them. Which is large.